### PR TITLE
New Blog on Project Governance

### PR DESCRIPTION
**Pull request by eeyun**
Signed-off-by: eeyun <ihenry@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by reset** [comment]
tenor-260573900

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+title: Habitat.sh Governance and Issue Tracking
+date: 2017-06-14
+author: Ian Henry
+tags: Governance,Community
+category: Community
+classes: body-article
+---
+Hopefully our previous [blog](https://www.habitat.sh/blog/2017/06/Triage-and-Contributions/) and [discourse]https://forums.habitat.sh/t/habitat-community-triage/287) posts have given you some general understanding around the way we govern the Habitat project. But, as a group of people hoping to share what we're building with a larger audience we know we can always be doing more to "pull back the curtain". We can be doing more to [expose our work](https://ext.prodpad.com/ext/roadmap/d2938aed0d0ad1dd62669583e108357efd53b3a6) and keep our community informed and educated on the ways we've chosen to operate.
+
+Thus, I'm here to explain as much as I can about the way we currently govern the Habitat Project and how the Issue Tracker fits in. Generally this blog post is going to build on top of the Triage and Contribution blog post linked
```
Comment (raskchanky) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: Missing a period at the end of the second sentence.

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Lots of new features might require community input. So, we suggest that the contributor whom submits the issue make sure to post about it in community channels to drum up user input. Contributors should also feel comfortable using the `@core-maintainers` slack notifier to notify the maintainers that the RFC has been opened.
+
+In the case of an RFC being opened. The core maintainers may request separate issues to be created for various aspects of the feature (dependent wholly on the size of the incoming work) which will be linked back to the original RFC. In the case that the RFC is a manageable chunk of work for an individual the issue will get sorted to Accepted Minor or Accepted Major milestones, the core engineering project board and the RFC prefix will be removed.
+
+## Merging Code
+* Open a Pull Request that includes a Developer Certificate of Origin-signed (DCO-signed) commit (anyone).
+* Code is reviewed by a Maintainer, Lieutenant, or Project Lead. Approval is indiciated by a thumbs up in a comment on the PR.
```
Comment (raskchanky) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: Sometimes that thumbs up comes in the form of a dank gif.  😁

**Comment by raskchanky** [comment]
tenor-125100857

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: Habitat.sh Governance and Issue Tracking
+date: 2017-06-14
+author: Ian Henry
+tags: Governance,Community
+category: Community
+classes: body-article
+---
+Hopefully our previous [blog](https://www.habitat.sh/blog/2017/06/Triage-and-Contributions/) and [discourse]https://forums.habitat.sh/t/habitat-community-triage/287) posts have given you some general understanding around the way we govern the Habitat project. But, as a group of people hoping to share what we're building with a larger audience we know we can always be doing more to "pull back the curtain". We can be doing more to [expose our work](https://ext.prodpad.com/ext/roadmap/d2938aed0d0ad1dd62669583e108357efd53b3a6) and keep our community informed and educated on the ways we've chosen to operate.
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]:
```
[discourse](https://forums.habitat.sh/t/habitat-community-triage/287)
``` 

(Missing an opening paren.)

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Thus, I'm here to explain as much as I can about the way we currently govern the Habitat Project and how the Issue Tracker fits in. Generally this blog post is going to build on top of the Triage and Contribution blog post linked
+
+## Core Team
+Habitat is governed by a core team, which is ultimately responsible for all decision-making in the project. Specifically, the core team alongside a Chef Inc. Product Owner:
+* Sets the overall direction and vision for the project
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: This bullet section needs to be offset by empty lines in order to get the formatting to work. (Right now it gets mashed into a single unformatted line.)

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* Sets the priorities and release schedule
+* Makes final decisions
+You can track the core maintainers in our [MAINTAINERS.md](https://github.com/habitat-sh/habitat/blob/master/MAINTAINERS.md) file at the root of the habitat-sh/habitat repository.
+
+## Tag Categorization
+We discussed this in [another blog post](https://www.habitat.sh/blog/2017/06/Triage-and-Contributions/) but it bares keeping in mind.
+| *TAGS Groups*                     |               Meaning                             |
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: Same thing here -- a preceding empty line gets the table to render.

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+Lots of new features might require community input. So, we suggest that the contributor whom submits the issue make sure to post about it in community channels to drum up user input. Contributors should also feel comfortable using the `@core-maintainers` slack notifier to notify the maintainers that the RFC has been opened.
+
+In the case of an RFC being opened. The core maintainers may request separate issues to be created for various aspects of the feature (dependent wholly on the size of the incoming work) which will be linked back to the original RFC. In the case that the RFC is a manageable chunk of work for an individual the issue will get sorted to Accepted Minor or Accepted Major milestones, the core engineering project board and the RFC prefix will be removed.
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: `... being opened, the core ...`

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+Any Maintainer may downvote on a PR, which increases the requirement for a patch to be merged to an absolute majority of Maintainers for the affected component(s), unless that Maintainer later changes their vote.
+
+## Patch Appeals Process
+
+There may be cases where someone wishes to appeal a Maintainer decision. In this event, the "chain of command" for the appeals process is as follows.
+
+    In the event that the actions of a Maintainer are to be appealed, the appeal should be directed to the Lieutenant for that component. As stated above, a Lieutenant retains veto power for the component(s) for which they are responsible.
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: These two blocks are being formatted as code, which is causing them to be rendered on a single side-scrolling line. Blockquotes (`>`) might make them a little more readable.

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+    In the event that the actions of a Lieutenant are to be appealed, the appeal should be directed to the Project Lead. As stated above, the Project Lead retains universal veto power over all components.
+
+Although Lieutenants and the Project Lead retain veto powers over certain components, use of this veto power is not guaranteed by the submission of an appeal to that person. It is expected that the majority decisions of component Maintainers and Lieutenants will be respected in all but the most exceptional circumstances.
+
+## Release Notes
+All merged PRs will be added into our release notes which are generated automatically when we cut a release. The Habitat product manager typically will aggregate that PR list into a post on the [forums](https://forums.habitat.sh/c/habitat-announcements) with any qualifying information in regards to new features that will be included in that release.
+
+
+This is the majority of the process around governance that the project has in place today. If youre interested in reading more make sure to keep track of the habitat wiki page on github, and check back here on our blog for updates in the future!
```
Comment (cnunciato) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: `If you're...`

File: www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md (hunk 1 above)
Comment (eeyun) [inline www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md]: indeed! thank you!

**Comment by eeyun** [comment]
I've cleaned up all of the responses thank you for all of your help!

**Comment by eeyun** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/243720729).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to pass the test suite](https://travis-ci.org/habitat-sh/habitat/builds/243720729).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/243720729) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by eeyun** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/243780188).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/243780188).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**